packages/adapter-postgres/src/index.ts

```diff
@@ -63,6 +72,36 @@ export const Adapter = createAdapter<TableTypes, FieldTypes, ConnectionConfig, C
     }
 
     const results = await db.query(select.toString());
+    const loads = context.loads;
+
+    if (loads) {
+      const loadPromises = Object.keys(loads).map(async (k) => {
```

Comment:
You can use `Object.entries()` to get both keys and values, then array destructure in the function sig.